centos-ci/libtask.sh

```diff
@@ -18,9 +18,6 @@ prepare_job() {
 
     . ~/rsync-password.sh
 
-    # Ensure we're operating on a clean base
-    (cd ${buildscriptsdir} && git clean -dfx && git reset --hard HEAD)
```

Comment:
Ah, OK, makes sense.  Hmm...I have this feeling though there was a reason I added that line.

Oh wait, I think I remember now.  The Duffy instances can be reused between jobs, and we don't use `rsync --delete` right now.